x-pack/plugins/observability_solution/observability/server/utils/queries.ts

```diff
@@ -83,6 +83,10 @@ export function rangeQuery(
   ];
 }
 
+export function existsQuery(field: string): QueryDslQueryContainer[] {
+  return [{ exists: { field } }];
+}
```

Comment:
ideally this should be moved to observability-shared

Reply:
I agree, felt weird to see this file here. 
I will keep this movement out of scope of this PR, as this file movement could trigger more changes